### ISSUE TITLE
show and let the user change emoji shortcut

### ISSUE THREAD

**Issue by eppixx** [elementary/switchboard-plug-keyboard#232]
Currently there is no indication of the emoji shortcut available in the keyboard settings. I knew about it from the Juno Blog post.
Also this means you can't change the shortcut that may be more accessable for other keyboard layouts.

**Comment by ryonakano** [comment]
Yeah we seemed to fix this in #500 which is available since OS 8.
